internal/version/version.go

```diff
@@ -2,14 +2,26 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-// Package version provides version information.
+// THIS FILE WAS AUTOMATICALLY GENERATED, PLEASE DO NOT EDIT.
+//
+// Generated on 2023-06-27T03:07:43Z by kres b93f606.
+
+// Package version contains variables such as project name, tag and sha. It's a proper alternative to using
+// -ldflags '-X ...'.
 package version
 
+import (
+	_ "embed"
+)
+
 var (
-	// Name is set at build time.
+	// Name declares project name.
+	//go:embed data/name
```

Comment:
Not sure if it makes sense to embed name.
The old code was setting name for each executable separately, so they all had their own unique names.

If we don't rely on that anymore, then probably it will make sense to set this name in the code somewhere.

Reply:
I agree, but how do me make a connection between the binary and the "line with the name" in the generated file? Previous version didn't need to do this, since it passed binary name during compilation of the said binary.

Reply:
Okay, I got the new implementation we should get the proper "per binary" name.